lib/src/main/java/graphql/nadel/NextgenEngine.kt

```diff
@@ -53,46 +56,49 @@ import kotlinx.coroutines.future.asCompletableFuture
 import kotlinx.coroutines.future.asDeferred
 import kotlinx.coroutines.future.await
 import kotlinx.coroutines.launch
-import java.util.Locale
 import java.util.concurrent.CompletableFuture
+import graphql.normalized.ExecutableNormalizedOperationFactory.Options.defaultOptions as executableNormalizedOperationFactoryOptions
 
-class NextgenEngine @JvmOverloads constructor(
-    nadel: Nadel,
+internal class NextgenEngine(
```

Comment:
Most of this is just changes to the constructor now that the engine abstraction is gone.

The `Nadel.Builder` now just passes everything required directly in here, so we don't need to go `nadel.` to get everything out.